pyccel/codegen/printing/ccode.py

```diff
@@ -1441,22 +1455,33 @@ def _print_AliasAssign(self, expr):
         lhs_var = expr.lhs
         rhs_var = expr.rhs
 
-        lhs = VariableAddress(lhs_var)
-        rhs = VariableAddress(rhs_var) if isinstance(rhs_var, Variable) else rhs_var
-
-        lhs = self._print(lhs)
-        rhs = self._print(rhs)
+        lhs_address = VariableAddress(lhs_var)
+        # Ensure all everything which can be stored in a VariableAddress is
```

Comment:
is this an incomplete comment or I didn't get the chance to understand it?

Reply:
There is a word to many. It should be "Ensure everything which can be stored in a VariableAddress is"

Reply:
To be understood as "Ensure everything which can be stored in a VariableAddress is stored in a VariableAddress"

Reply:
Fixed 4f09939